packages/react-charts/src/components/ChartArea/examples/ChartArea.md

```diff
@@ -173,7 +174,8 @@ class BottomAlignedLegend extends React.Component {
 }
 ```
 
-```js title=Multi--color-(unordered)-bottom--left-aligned-legend-and-responsive-container
+### Multi-color (unordered) bottom  left aligned legend and responsive container
```

Comment:
guessing the previous double dash was an escaped single dash? then this should say `bottom-left`